library/std/src/f32/tests.rs

```diff
@@ -550,6 +550,21 @@ fn test_atanh() {
     assert_approx_eq!((-0.5f32).atanh(), -0.54930614433405484569762261846126285f32);
 }
 
+#[test]
+fn test_gamma() {
+    assert_eq!(0.0f32.gamma(), f32::INFINITY);
```

Comment:
This test causes compile time errors on platforms like `x86_64-fortanix-unknown-sgx` and probably others:
```
12:04:56   = note: rust-lld: error: undefined symbol: tgammaf
12:04:56           >>> referenced by std.143fe29c-cgu.2
12:04:56           >>>               /home/jenkins/workspace/rust-sgx-ci/rust/build/x86_64-unknown-linux-gnu/stage1-std/x86_64-fortanix-unknown-sgx/release/deps/std-e08c880d8c834560.std.143fe29c-cgu.2.rcgu.o:(std::f32::tests::test_gamma::he9b80e8a0cc72b6a)
12:04:56           >>> referenced by std.143fe29c-cgu.2
12:04:56           >>>               /home/jenkins/workspace/rust-sgx-ci/rust/build/x86_64-unknown-linux-gnu/stage1-std/x86_64-fortanix-unknown-sgx/release/deps/std-e08c880d8c834560.std.143fe29c-cgu.2.rcgu.o:(std::f32::tests::test_gamma::he9b80e8a0cc72b6a)
12:04:56           >>> referenced by std.143fe29c-cgu.2
12:04:56           >>>               /home/jenkins/workspace/rust-sgx-ci/rust/build/x86_64-unknown-linux-gnu/stage1-std/x86_64-fortanix-unknown-sgx/release/deps/std-e08c880d8c834560.std.143fe29c-cgu.2.rcgu.o:(std::f32::tests::test_gamma::he9b80e8a0cc72b6a)
12:04:56           >>> referenced 6 more times
12:04:56           
12:04:56           rust-lld: error: undefined symbol: tgamma
12:04:56           >>> referenced by std.143fe29c-cgu.3
12:04:56           >>>               /home/jenkins/workspace/rust-sgx-ci/rust/build/x86_64-unknown-linux-gnu/stage1-std/x86_64-fortanix-unknown-sgx/release/deps/std-e08c880d8c834560.std.143fe29c-cgu.3.rcgu.o:(std::f64::tests::test_gamma::hff115a05914e5bdc)
12:04:56           >>> referenced by std.143fe29c-cgu.3
12:04:56           >>>               /home/jenkins/workspace/rust-sgx-ci/rust/build/x86_64-unknown-linux-gnu/stage1-std/x86_64-fortanix-unknown-sgx/release/deps/std-e08c880d8c834560.std.143fe29c-cgu.3.rcgu.o:(std::f64::tests::test_gamma::hff115a05914e5bdc)
12:04:56           >>> referenced by std.143fe29c-cgu.3
12:04:56           >>>               /home/jenkins/workspace/rust-sgx-ci/rust/build/x86_64-unknown-linux-gnu/stage1-std/x86_64-fortanix-unknown-sgx/release/deps/std-e08c880d8c834560.std.143fe29c-cgu.3.rcgu.o:(std::f64::tests::test_gamma::hff115a05914e5bdc)
12:04:56           >>> referenced 6 more times
12:04:56           
12:04:56 
12:04:56 error: could not compile `std` due to previous error
```
Looking into it, these `tgamma` and `tgammaf` symbols exists, but are not exported from the `compiler-builtins` crate. These should probably be added [here](https://github.com/rust-lang/compiler-builtins/blob/master/src/math.rs#L17-L66).

Reply:
Thanks @raoulstrackx! Submitted a PR there: https://github.com/rust-lang/compiler-builtins/pull/482

Reply:
Great! Thanks @ankane !